src/sentry/models/sentryfunction.py

```diff
@@ -4,6 +4,15 @@
 from sentry.db.models.fields.array import ArrayField
 
 
+class SentryFunctionManager(BaseManager):
+    def get_sentry_functions(self, organization_id, event_type):
+        # TODO: check events
+        functions = self.filter(
+            organization_id=organization_id,
+        )
+        return filter(lambda fn: event_type in fn.events, list(functions))
```

Comment:
Are you not able to apply this condition in SQL? Django has a `__contains` lookup. You could do `events__contains=event_type` in your conditions.